src/symlink.c

```diff
@@ -40,6 +40,8 @@ int main(int argc, char * const *argv)
 	else
 	if ((name = strrchr(strcpy(path, argv[0]), '/')))
 		name++;
+	else if ((name = strrchr(strcpy(path, argv[0]), '\\')))
```

Comment:
The second `strcpy()` here should be unnecessary. So:

```c
	else if ((name = strrchr(path, '\\')))
```

We assume that a `strcpy()` like this was reached when we do `name = path;` in the `else` code path below anyway.

Reply:
You could even shorten this further to:

```c
	if (!argv[0])
		name = path;
	else if ((name = strrchr(strcpy(path, argv[0]), '/')) || (name = strrchr(path, '\\')))
		name++;
	else
		name = path;
```

relying on C's guaranteed short-circuit evaluation.